models/lioncore.json

```diff
@@ -1,661 +1,661 @@
 [
   {
-    "type": "jj0Y6KYlHt3_qyFDFAx4CA",
-    "id": "31_xAp4rmNVj6Sdahgjeuw",
+    "type": "LIonCore_M3_Metamodel",
+    "id": "LIonCore_M3",
     "properties": {
-      "97hsg4Fg3ZitV5FaapRLVg": "LIonCore"
+      "LIonCore_M3_Metamodel_qualifiedName": "LIonCore.M3"
```

Comment:
is the dot on the right-hand side on purpose?

Reply:
Yup: I thought the dot was OK for use in qualified name fields, and subsequently qualified names get their dots replaced by underscores to get their IDs.